lib/bamboo/adapters/mandrill_adapter.ex

```diff
@@ -91,8 +92,16 @@ defmodule Bamboo.MandrillAdapter do
 
   defp convert_to_mandrill_params(email, api_key) do
     %{key: api_key, message: message_params(email)}
+    |> add_template_params(email)
```

Comment:
What do you think of calling this `maybe_put_template_params`. That way it's clear that it sometimes won't add the params